README.md

```diff
@@ -27,53 +22,196 @@ ember install ember-a11y-testing
+import { customInvocationStrategy } from './custom-invocation-strategy';
+
+setApplication(Application.create(config.APP));
+
+setupGlobalA11yHooks(customInvocationStrategy);
+
+start();
+```
+
+Examples of invocation strategies can be found in [this](https://github.com/ember-a11y/ember-a11y-testing/blob/50ef5f8fff4aa91d7a85b9feee5b1ce9bf380df9/tests/acceptance/setup-global-a11y-hooks-test.ts#L32) [repository's](https://github.com/ember-a11y/ember-a11y-testing/blob/50ef5f8fff4aa91d7a85b9feee5b1ce9bf380df9/tests/acceptance/setup-global-a11y-hooks-test.ts#L81) [tests](https://github.com/ember-a11y/ember-a11y-testing/blob/50ef5f8fff4aa91d7a85b9feee5b1ce9bf380df9/tests/acceptance/setup-global-a11y-hooks-test.ts#L135).
+
+#### Setting Options with `setupGlobalA11yHooks`
+
+You can provide options to axe-core for your tests using the `setRunOptions` API. Options can
+be set a few ways:
+
+Globally:
+
+```javascript
+// tests/test-helper.js
+import { setRunOptions } from 'ember-a11y-testing';
+
+setRunOptions({
+  rules: {
+    region: { enabled: true },
+  },
+  checks: {
+    'color-contrast': {
+      options: {
+        noScroll: true,
+      },
+    },
+  },
+});
+```
+
+Test module level:
 
-Ember A11y Testing also provides a simple helper to run accessibility audits
-on-demand within your test suite.
+```javascript
+import { module, test } from 'qunit';
+import { setRunOptions } from 'ember-a11y-testing';
 
-_Note:_ any tests run with Ember A11y Testing will adjust the testing container
+module('some test module', function (hooks) {
+  hooks.beforeEach(function () {
+    setRunOptions({
+      rules: {
+        region: { enabled: true },
+      },
+      checks: {
+        'color-contrast': {
+          options: {
+            noScroll: true,
+          },
+        },
+      },
+    });
+  });
+
+  // ...
+});
+```
+
+Individual test level:
+
+```javascript
+import { module, test } from 'qunit';
+import { a11yAudit } from 'ember-a11y-testing/test-support';
+import { setRunOptions } from 'ember-a11y-testing';
+
+module('some test module', function (hooks) {
+  test('some test', function (assert) {
+    setRunOptions({
+      rules: {
+        region: { enabled: true },
+      },
+      checks: {
+        'color-contrast': {
+          options: {
+            noScroll: true,
+          },
+        },
+      },
+    });
+
+    // ...
+    a11yAudit();
+    // ...
+  });
+
+  // ...
+});
+```
+
+When using `setRunOptions` during a test, the options you set are automatically reset when the test completes.
+
+### `a11yAudit` Usage
+
+`ember-a11y-testing` also provides a test helper to run accessibility audits
+on specific tests within your test suite.
+
+_Note:_ any tests run with `ember-a11y-testing` will adjust the testing container
 to occupy the entire screen. This is to simulate the actual application
 environment, as browsers adjust styles at small sizes for accessibility reasons.
 It will reset itself at the conclusion of testing though.
```

Comment:
Do we actually do this?

Reply:
No...

Reply:
It was in the original docs. It struck me as weird, since I didn't see any code that would do this. I'll remove it.